packages/components/src/styles/components/dropdown.scss

```diff
@@ -79,7 +79,7 @@ $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
   display: flex;
   align-items: center;
   justify-content: center;
-  border-radius: 3px; // $hds-dropdown-toggle-border-radius - 1px padding - 1px border
+  border-radius: var(--token-border-radius-x-small); // $hds-dropdown-toggle-border-radius - 1px padding - 1px border
```

Comment:
[thought] personally I would not use a token here; visually in Figma the border radius may not be 3px; here is computed, and I would leave it as a pure number, to stress the fact that it is computed (alternatively, we could consider using a `calc()` instead, here)

Reply:
Makes sense, thanks

Reply:
Updated